rules.yaml

```diff
@@ -2,35 +2,41 @@ features:
   - name: api
     scopes:
       - match: module
-        prefix: "modules/{{snake .Module.Name}}_module/{{ snake .Module.Name}}/"
         documents:
+          - source: CMakeLists.txt.tpl
+            target: CMakeLists.txt
```

Comment:
Every module overwrites this file. This should be on `match: system` level.
If you check the Goldenmaster CMakeLists.txt it only contains the last module and not a list to all.